tests/model/test_model.py

```diff
@@ -2882,7 +2882,7 @@ def test_toggle_stream_visual_notifications(
                     "id": 0,
                 },
                 False,
-                {"sender_name": "hamlet"},
+                {"hamlet"},
```

Comment:
In the cases of 1. `in_pm_narrow_with_sender_typing:start`, 
2. `in_pm_narrow_with_sender_typing:start_while_animation_in_progress`,
3 `in_pm_narrow_with_sender_typing:stop`,
when the test function calls `model._handle_typing_event(event)`, there is a AttributeError where `self.controller.active_conversation_info` is considered as `Dict` where as it is actually `Set`